lte/gateway/docker/.env

```diff
@@ -18,7 +18,6 @@ OPTIONAL_ARCH_POSTFIX=
 
 BUILD_CONTEXT=../../..
 
-ROOTCA_PATH=../../../.cache/test_certs/rootCA.pem
```

Comment:
Are we not using this path for the development?

Reply:
Good point. But we can probably just edit this file and adapt `CERTS_VOLUME` when running this locally on the magma-dev VM.

Reply:
The problem is that in a production environment, the `.cache` path does not exist, so docker throws an error when trying to mount it. As the certificate already exists in `/var/opt/magma/certs/rootCA.pem` in the container (it was mounted before as part of `CERTS_VOLUME`), docker tries to create a directory where there is already a file present. The error is something like this:

Error response from daemon: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: error mounting "/root/workspace/magma/.cache/test_certs/rootCA.pem" to rootfs at "/var/opt/magma/certs/rootCA.pem": mount /root/workspace/magma/.cache/test_certs/rootCA.pem:/var/opt/magma/certs/rootCA.pem (via /proc/self/fd/6), flags: 0x5000: not a directory: unknown: Are you trying to mount a directory onto a file (or vice-versa)? Check if the specified host path exists and is the expected type

I will try to find a better solution, if the path is needed for development.

Reply:
Thanks @wolfseb

Reply:
> I will try to find a better solution, if the path is needed for development.

You can probably just copy the certificate to the right location before launching the containers. Running the containers already requires some copying of files, see https://github.com/magma/magma/tree/master/lte/gateway/docker#running-the-containerized-agw-locally-on-the-magma-vm

Reply:
Thanks for the tip, done.